### ISSUE TITLE
Error parsing zpool status output

### ISSUE THREAD

**Issue by asw44asw** [damicon/zfswatcher#20]
Hello Developers.
I hope contribute with the code refinement 
I had an error when starting zfswatcher,
The log file explains by itself
Thank you all
<pre>
2020-10-01 20:38:17 [info] zfswatcher starting
2020-10-01 20:38:17 [crit] invalid line 11 in status output: remove: Removal of vdev 0 copied 552G in 2h27m, completed on Mon Aug 24 05:40:45 2020
>  pool: backup_servers
> state: ONLINE
>status: One or more devices has experienced an unrecoverable error.  An
>       attempt was made to correct the error.  Applications are unaffected.
>action: Determine if the device needs to be replaced, and clear the errors
>       using 'zpool clear' or replace the device with 'zpool replace'.
>   see: http://zfsonlinux.org/msg/ZFS-8000-9P
>  scan: scrub in progress since Thu Oct  1 19:03:39 2020
>       315G scanned at 56.8M/s, 215G issued at 38.8M/s, 716G total
>       36K repaired, 30.02% done, 0 days 03:40:32 to go
>remove: Removal of vdev 0 copied 552G in 2h27m, completed on Mon Aug 24 05:40:45 2020
>    7.21M memory used for removed device mappings
>config:
>
>       NAME                                    STATE     READ WRITE CKSUM
>       backup_servers                          ONLINE       0     0     0
>         mirror-1                              ONLINE       0     0     0
>           ata-ST1000DM003-1CH162_S1DGGSXD     ONLINE       0     0     0
>           ata-SAMSUNG_HD103SJ_S2NWJ50B351535  ONLINE       1     0     3  (repairing)
>
>errors: No known data errors
>
>  pool: dpool
> state: ONLINE
>  scan: scrub in progress since Thu Oct  1 19:03:49 2020
>       308G scanned at 55.6M/s, 288G issued at 52.1M/s, 388G total
>       0B repaired, 74.37% done, 0 days 00:32:33 to go
>config:
>
>       NAME                                          STATE     READ WRITE CKSUM
>       dpool                                         ONLINE       0     0     0
>         mirror-0                                    ONLINE       0     0     0
>           ata-SAMSUNG_HD154UI_S240J50B101993-part3  ONLINE       0     0     0
>           ata-SAMSUNG_HD154UI_S240J50B712198-part3  ONLINE       0     0     0
>
>errors: No known data errors
>
>  pool: vpool
> state: ONLINE
>  scan: scrub in progress since Thu Oct  1 19:03:56 2020
>       938G scanned at 170M/s, 655G issued at 118M/s, 2.14T total
>       0B repaired, 29.92% done, 0 days 03:40:56 to go
>config:
>
>       NAME                               STATE     READ WRITE CKSUM
>       vpool                              ONLINE       0     0     0
>         ata-ST3000VX000-1ES166_Z501YC0L  ONLINE       0     0     0
>       
>errors: No known data errors
2020-10-01 20:38:17 [crit] exiting, parsing ZFS status failed
2020-10-01 20:38:17 [info] zfswatcher stopping
</pre>
And the zpool status output
<pre>
# zpool status
  pool: backup_servers
 state: ONLINE
status: One or more devices has experienced an unrecoverable error.  An
	attempt was made to correct the error.  Applications are unaffected.
action: Determine if the device needs to be replaced, and clear the errors
	using 'zpool clear' or replace the device with 'zpool replace'.
   see: http://zfsonlinux.org/msg/ZFS-8000-9P
  scan: scrub in progress since Thu Oct  1 19:03:39 2020
	352G scanned at 51,5M/s, 252G issued at 36,9M/s, 716G total
	36K repaired, 35,22% done, 0 days 03:34:53 to go
remove: Removal of vdev 0 copied 552G in 2h27m, completed on Mon Aug 24 05:40:45 2020
    7,21M memory used for removed device mappings
config:

	NAME                                    STATE     READ WRITE CKSUM
	backup_servers                          ONLINE       0     0     0
	  mirror-1                              ONLINE       0     0     0
	    ata-ST1000DM003-1CH162_S1DGGSXD     ONLINE       0     0     0
	    ata-SAMSUNG_HD103SJ_S2NWJ50B351535  ONLINE       1     0     3  (repairing)

errors: No known data errors

  pool: dpool
 state: ONLINE
  scan: scrub in progress since Thu Oct  1 19:03:49 2020
	356G scanned at 52,0M/s, 337G issued at 49,3M/s, 388G total
	0B repaired, 86,96% done, 0 days 00:17:29 to go
config:

	NAME                                          STATE     READ WRITE CKSUM
	dpool                                         ONLINE       0     0     0
	  mirror-0                                    ONLINE       0     0     0
	    ata-SAMSUNG_HD154UI_S240J50B101993-part3  ONLINE       0     0     0
	    ata-SAMSUNG_HD154UI_S240J50B712198-part3  ONLINE       0     0     0

errors: No known data errors

  pool: vpool
 state: ONLINE
  scan: scrub in progress since Thu Oct  1 19:03:56 2020
	1,03T scanned at 154M/s, 793G issued at 116M/s, 2,14T total
	0B repaired, 36,26% done, 0 days 03:24:51 to go
config:

	NAME                               STATE     READ WRITE CKSUM
	vpool                              ONLINE       0     0     0
	  ata-ST3000VX000-1ES166_Z501YC0L  ONLINE       0     0     0

errors: No known data errors
</pre>

**Comment by mariomare22** [comment]
having the same issue here any luck?

**Comment by asw44asw** [comment]
No, I gave up using zfswatcher and did my own script in bash, running 
from cron, based in the return from "zpool status -x" command.

Em 23/02/2021 14:38, mariomare22 escreveu:
